staging/src/k8s.io/csi-translation-lib/plugins/azure_disk_test.go

```diff
@@ -290,9 +294,9 @@ func TestTranslateTranslateCSIPVToInTree(t *testing.T) {
 							FSType:   "fstype",
 							ReadOnly: true,
 							VolumeAttributes: map[string]string{
-								azureDiskCachingMode: "cachingmode",
-								azureDiskFSType:      fsType,
-								azureDiskKind:        "managed",
+								"cachingmode":   "ReadOnly",
```

Comment:
Can you add a test case using the camelcase field names? It looks like the older version of the driver accepted camelcased fields: https://github.com/kubernetes-sigs/azuredisk-csi-driver/blob/release-1.0/pkg/azuredisk/controllerserver.go#L907

Also, on a related note, it looks like the latest version of the driver no longer accepts camelcase fields. That appears to be a backwards incompatible change that you probably want to fix. https://github.com/kubernetes-sigs/azuredisk-csi-driver/blob/master/pkg/azureutils/azure_disk_utils.go#L126:6

Reply:
thanks!